apps/docs/src/components/Calendar.tsx

```diff
@@ -0,0 +1,41 @@
+'use client'
```

Comment:
Is it possible to remove this and render the component server-side? Then we can move all of the dependencies to dev dependencies, and the bundle size will be much smaller for the client. 

What would be the other tradeoffs? Does this calendar update real-time?

Reply:
I removed it and it gave me this error:
![image](https://github.com/user-attachments/assets/089d50a0-0967-4003-9865-300d7e2e382c)
I did some research online and there doesn't seem to be a way to render it server-side, at least as a React component. I think it needs to be client side in order to be interactive.

Reply:
Okay sounds good